modules/stanford_profile_styles/lib/scss/component/paragraph/lists/index.scss

```diff
@@ -240,34 +240,74 @@
 }
```

Comment:
all of the opportunity node styles should be in the `stanford_opportunity.scss`. These styles do not load when we use the filtered list paragraph. We will have to create a new `paragraph/filtered_lists/index.scss` styles that will be used to style the layout of the view and the filters/buttons/pagers. But it won't be used to style the display of the node card or the node teaser.

Reply:
Ok. I removed all the opp list to the `stanford_opportunity.scss`. I also fixed the classes so the teaser and card are getting the same styles.  Let me know what you think.  thank you for the feedback.